src/main/java/com/ita/if103java/ims/mapper/jdbc/EventRowMapper.java

```diff
@@ -16,12 +18,12 @@ public Event mapRow(ResultSet resultSet, int i) throws SQLException {
         Event event = new Event();
         event.setId(resultSet.getLong("id"));
         event.setMessage(resultSet.getString("message"));
-        event.setDate(resultSet.getObject("date", ZonedDateTime.class));
+        event.setDate(ZonedDateTime.of(resultSet.getObject("date", LocalDateTime.class), ZoneId.systemDefault()));
```

Comment:
Does it not work with ZoneDateTime?

Reply:
I had tried, but I couldn't insert ZonedDateTime into table